### PR TITLE
lexicon: 3.4.3 -> 3.4.4

### PR DESCRIPTION
**Pull request by flyfloh**
Update lexicon to  current version.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [X] macOS
   - [ ] other Linux distributions
- [X] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [X] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [X] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 102223` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>lexicon</li>
  </ul>
</details>

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 102223` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>lexicon</li>
  </ul>
</details>

File: pkgs/tools/admin/lexicon/default.nix [hunk 1]
```diff
@@ -1,44 +1,20 @@
...
-      });
-
-      localzone = super.localzone.overridePythonAttrs(oldAttrs: rec {
-        meta = oldAttrs.meta // { broken = false; };
-      });
-    };
-  };
-in
-  with py.pkgs;
+with python3Packages;
```
Comment (SuperSandro2000) [inline pkgs/tools/admin/lexicon/default.nix:6]: Please don't use with in the entire file.

Comment (flyfloh) [reply pkgs/tools/admin/lexicon/default.nix]: Fixed

**Comment by SuperSandro2000** [comment]
@flyfloh Please resolve the merge conflict.

**Comment by flyfloh** [comment]
`master` has been updated to 3.5 already. So this is not necessary any more.

**Pull request closed without merging**